tests/unit/s2n_connection_test.c

```diff
@@ -126,7 +126,7 @@ int main(int argc, char **argv)
      */
     {
         /* Carefully consider any increases to this number. */
-        const uint16_t max_connection_size = 4190;
+        const uint16_t max_connection_size = 4250;
```

Comment:
You're just bumping this to 4250 so we stop bumping it for single pointers, right? You didn't actually add 60 bytes?
Although: why does this change cause a memory increase at all?

Reply:
Sorry, this was here because I added an extra byte to the array struct, but it's no longer needed. I removed it.